integrationTests/vm/txsFee/asyncCall_test.go

```diff
@@ -70,3 +73,55 @@ func TestAsyncCallShouldWork(t *testing.T) {
 	require.Equal(t, big.NewInt(50000000), testContext.TxFeeHandler.GetAccumulatedFees())
 	require.Equal(t, big.NewInt(4999988), testContext.TxFeeHandler.GetDeveloperFees())
 }
+
+func TestMinterContractWithAsyncCalls(t *testing.T) {
+	testContext, err := vm.CreatePreparedTxProcessorWithVMs(config.EnableEpochs{})
```

Comment:
can you replace this call with the following?
```GO
	testContext, err := vm.CreatePreparedTxProcessorWithVMsAndCustomGasSchedule(config.EnableEpochs{}, func(gasMap arwenConfig.GasScheduleMap) {
		gasMap[common.MaxPerTransaction]["MaxBuiltInCallsPerTx"] = 140
		gasMap[common.MaxPerTransaction]["MaxNumberOfTransfersPerTx"] = 140
		gasMap[common.MaxPerTransaction]["MaxNumberOfTrieReadsPerTx"] = 140
	})
```

Reply:
done